snet_cli/mpe_service_metadata.py

```diff
@@ -0,0 +1,101 @@
+# functions for manipulate with server.json file
+# metadata format:
+#----------------------------------------------------
+# version         - used to track format changes (current version is 1)
+# model_ipfs_hash - IPFS HASH to the .tar archive of protobuf service specification
+# mpe_address     - Address of MultiPartyEscrow contract. 
+#                   Client should use it exclusively for cross-checking of mpe_address, 
+#                        (because service can attack via mpe_address)
+#                   Daemon can use it directly if authenticity of metadata is confirmed
+# pricing {}      - Pricing model
+#          Possible pricing models:
+#          1. Fixed price
+#              price_model  - "fixed_price"
+#              price        -  unique fixed price for all method
+#              (other pricing models can be easily supported)
+# groups[]       - group is the number of endpoints which shares same payment channel; 
+#                   grouping strategy is defined by service provider; 
+#                   for example service provider can use region name as group id
```

Comment:
```suggestion
#                   for example service provider can use region name as group name
```

Reply:
Yes. I've changed it also here https://github.com/singnet/wiki/blob/master/multiPartyEscrowContract/MPEServiceMetadata.md